src/toastify.js

```diff
@@ -116,6 +117,31 @@
           }.bind(this)
         );
 
+        // Reset timeout if toats is focused
```

Comment:
Should read "toast" 😉 Maybe you can rephrase it to "Clear timeout while toast is focused"? Seems more fitting to what the function does.

Otherwise it looks good.

Reply:
Indeed! :see_no_evil: